style.css

```diff
@@ -1,6 +1,74 @@
-@media (min-width:320px) { /* smartphones, iPhone, portrait 480x320 phones */ }
-@media (min-width:481px) { /* portrait e-readers (Nook/Kindle), smaller tablets @ 600 or @ 640 wide. */ }
-@media (min-width:641px) { /* portrait tablets, portrait iPad, landscape e-readers, landscape 800x480 or 854x480 phones */ }
-@media (min-width:961px) { /* tablet, landscape iPad, lo-res laptops ands desktops */ }
-@media (min-width:1025px) { /* big landscape tablets, laptops, and desktops */ }
-@media (min-width:1281px) { /* hi-res laptops and desktops */ }
\ No newline at end of file
+.body__text  {
+	background: blue;
+	color: white;
+	text-align: center;
+	min-width: 280px;
+	max-width: 700px;
+	white-space: pre-line;	
+}
+
+/* smartphones, iPhone, portrait 480x320 phones */
+
+@media (min-width:320px) {
+  .body__title  {
+  	font-size: 20px;
+  }
+  .body__text  {
+  	font-size: 12px;
+  }
+}
+
+/* portrait e-readers (Nook/Kindle), smaller tablets @ 600 or @ 640 wide. */
+
+@media (min-width:481px) {
+  .body__title  {
+  	font-size: 30px;
+  }
+  .body__text  {
+  	font-size: 18px;
+  }
+}
+
+/* portrait tablets, portrait iPad, landscape e-readers, landscape 800x480 or 854x480 phones */
+
+@media (min-width:641px) {
+  .body__title  {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/media_adaptive/pullRequest?prid=5498105)